syntax="proto3";

message hgAlgorOwnShip
{
    repeated string MMSI = 2;
}